### PR TITLE
Add callout boxes to #119

### PR DESCRIPTION
**Pull request by dafnevk**
This pull request will add several callout boxes to adress #119 
- [ ] Size of training data needed
- [ x] How to decide on the number of layers/nodes?
- [ x] How do I search for pre-trained models or existing architectures?
- [ ] Explainable AI: How do you know what the neural network is responding to?
- [ ] Why do we use "relu" as activation function?
- [ ] Is Dropout (random silencing of nodes) applied per epoch or per batch?
- [ ] Is it necessary to remove co-linear (or highly correlated) features?

### CODE REVIEW

**Comment by dafnevk** [comment]
I added a remark about dropout per training instance in the main text instead of a call out box, because I think it is useful to highlight this anyway and it does not require too much explanation.

I'm leaving the last two items for separate issues:
- activation functions / relu: #101 
- explainable AI: will make issue for this

**Comment by psteinb** [comment]
This is not marked as a draft, but 2 tasks are outstanding. I'll go ahead and review what is there, so that the material will get merged.

**Comment by dafnevk** [comment]
> This is not marked as a draft, but 2 tasks are outstanding. I'll go ahead and review what is there, so that the material will get merged.

Thank you, see my remark above why the two boxes are not checked, I created separate issues for that (I guess that means I could also check those boxes).
